samples/create-training-pipeline-image-classification.js

```diff
@@ -0,0 +1,134 @@
+  const ModelType = definition.AutoMlImageClassificationInputs.ModelType;
+
+  // Specifies the location of the api endpoint
+  const clientOptions = {
+    apiEndpoint: 'us-central1-aiplatform.googleapis.com',
+  };
+
+  // Instantiates a client
+  const pipelineServiceClient = new aiplatform.PipelineServiceClient(
+    clientOptions
+  );
+
+  async function createTrainingPipelineImageClassification() {
+    // Configure the parent resource
+    const parent = `projects/${project}/locations/${location}`;
+
+    // Values should match the input expected by your model.
+    const trainingTaskInputsMessage = new definition.AutoMlImageClassificationInputs(
+      {
+        multiLabel: true,
+        modelType: ModelType.CLOUD,
+        budgetMilliNodeHours: 8000,
+        disableEarlyStopping: false,
+      }
+    );
+
+    const trainingTaskInputs = trainingTaskInputsMessage.toValue();
+
+    const trainingTaskDefinition =
+      'gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_image_classification_1.0.0.yaml';
+
+    const modelToUpload = {displayName: modelDisplayName};
+    const inputDataConfig = {datasetId: datasetId};
+    const trainingPipeline = {
+      displayName: trainingPipelineDisplayName,
+      trainingTaskDefinition,
+      trainingTaskInputs,
+      inputDataConfig: inputDataConfig,
+      modelToUpload: modelToUpload,
+    };
+    const request = {
+      parent,
+      trainingPipeline,
+    };
+
+    console.log(request);
+
+    // Create training pipeline request
+    const [response] = await pipelineServiceClient.createTrainingPipeline(
+      request
+    );
+
+    console.log('Create training pipeline image classification response');
+    console.log(`\tName : ${response.name}`);
+    console.log(`\tDisplay Name : ${response.displayName}`);
+    console.log(
+      `\tTraining task definition : ${response.trainingTaskDefinition}`
+    );
+    console.log(
+      `\tTraining task inputs : \
+        ${JSON.stringify(response.trainingTaskInputs)}`
+    );
+    console.log(
+      `\tTraining task metadata : \
+        ${JSON.stringify(response.trainingTaskMetadata)}`
+    );
+    console.log(`\tState ; ${response.state}`);
+    console.log(`\tCreate time : ${JSON.stringify(response.createTime)}`);
+    console.log(`\tStart time : ${JSON.stringify(response.startTime)}`);
+    console.log(`\tEnd time : ${JSON.stringify(response.endTime)}`);
+    console.log(`\tUpdate time : ${JSON.stringify(response.updateTime)}`);
+    console.log(`\tLabels : ${JSON.stringify(response.labels)}`);
+
+    const error = response.error;
+    console.log('\tError');
+    if (error === null) {
+      console.log('\t\tCode : {}');
+      console.log('\t\tMessage : {}');
+    } else {
+      console.log(`\t\tCode : ${error.code}`);
+      console.log(`\t\tMessage : ${error.message}`);
+    }
+  }
+  // [END aiplatform_create_training_pipeline_image_classification]
+  await createTrainingPipelineImageClassification();
```

Comment:
@telpirion if you take this approach, then the `await` should be dropped. @sofisl @fhinkel, it would be good to update the sample policies to include `uncaughtRejections`, so that we properly exit with status code `1`, even when we're not awaiting.

Reply:
Done.